kitsune/sumo/static/sumo/js/wiki.js

```diff
@@ -787,4 +787,20 @@
     });
   }
 
+  var draftbutton = $('.btn-draft');
+  var url = $('.btn-draft' ).data( 'draft-url' );
+  var draftmessage = $('.draft-message').html();
+  draftbutton[0].addEventListener('click', function() {
```

Comment:
jQuery has a built-in `.click` [method we can use](https://api.jquery.com/click/) instead of `addEventListener`. This will remove the need for the `[0]` and clean up the syntax a little bit. So this could be re-written like:

```javascript
$draftButton.click(function() {
   // do things
});
```